react/src/lib/components/SubsurfaceViewer/components/DistanceScale.tsx

```diff
@@ -28,7 +28,10 @@ const DistanceScale: React.FC<ScaleProps> = ({
     style,
     scaleUnit,
 }: ScaleProps) => {
-    if (!zoom || !widthPerUnit || !incrementValue) return null;
+    // @rmt: added scaleUnit check - NOTE: if any of the values below === 0 || === "", this will return null
+    if (!zoom || !widthPerUnit || !incrementValue || scaleUnit === undefined) {
```

Comment:
Can't you test `convert(0).getUnit(scaleUnit)` to make sure the unit does indeed exist ?

If scaleUnits is not valid, the conversion will throw :/